lib/otel/attr-mapping/http.js

```diff
@@ -83,8 +83,15 @@ const attrMappings = {
     attrs: [constants.ATTR_HTTP_ROUTE],
     mapping({ segment, transaction }) {
       return (value) => {
-        transaction.nameState.appendPath(value)
-        segment.addAttribute('http.route', value)
+        if (value.length > 0) {
+          transaction.nameState.appendPath(value)
+          segment.addAttribute('http.route', value)
+        } else if (segment.attributes['http.target']) {
```

Comment:
this is not the intent of the mapping function. it lists all otel span attributes that map to what we need. `http.target` is not a precursor to `http.route`.  Let's sync up to discuss the right way to solve this